inductiva/tasks/streams.py

```diff
@@ -268,6 +275,9 @@ def _run_websocket(self):
         and update the start query to get the logs from the last
         received timestamp."""
 
+        for sig in [signal.SIGTERM, signal.SIGINT, signal.SIGHUP]:
+            signal.signal(sig, self._handle_termination)
```

Comment:
Why don't we just do 
```
if self._end_of_stream or self._keyboard_interrupt:
      self._disable_logs()
      break
```
I don't think there is a way to capture a sigkill.
@sergiomsantos can give a better input on this.